pkg/ext-proc/main.go

```diff
@@ -144,53 +157,103 @@ func main() {
 		Zone:           *zone,
 		ServerPoolName: *serverPoolName,
 	}).SetupWithManager(mgr); err != nil {
-		klog.Error(err, "Error setting up EndpointSliceReconciler")
+		klog.Fatalf("Error setting up EndpointSliceReconciler: %v", err)
 	}
 
-	errChan := make(chan error)
+	// Start the controller manager
+	wg.Add(1)
```

Comment:
can we encapsulate each goroutine in a function prefixed with `Start`? 
- StartSignalHandler
- StartHealthServer
- StartExternalProcessorServer 
-